code/modules/mob/living/living.dm

```diff
@@ -293,9 +293,14 @@
 
 		log_combat(src, M, "grabbed", addition="passive grab")
 		if(!supress_message && !(iscarbon(AM) && HAS_TRAIT(src, TRAIT_STRONG_GRABBER)))
-			M.visible_message("<span class='warning'>[src] grabs [M] [(zone_selected == "l_arm" || zone_selected == "r_arm")? "by their hands":"passively"]!</span>", \
-							"<span class='warning'>[src] grabs you [(zone_selected == "l_arm" || zone_selected == "r_arm")? "by your hands":"passively"]!</span>", null, null, src)
-			to_chat(src, "<span class='notice'>You grab [M] [(zone_selected == "l_arm" || zone_selected == "r_arm")? "by their hands":"passively"]!</span>")
+			var/mob/living/L = M
+			if (L.getorgan(/obj/item/organ/tail/cat) && zone_selected == BODY_ZONE_PRECISE_GROIN)
+				M.visible_message("<span class ='warning'> [src] grabs [L] by the tail!</span>", null, null, src)
+				to_chat(src, "<span class='notice'>You grab [L] by their tail! Weirdo!</span>")
```

Comment:
This should be merged into 1 line and use the proper functionality of visible_message

Reply:
Tried, doesnt work. Did add a missing string, though